### DISCUSSION TITLE
Copilot is inactive

### DISCUSSION THREAD

**Discussion by Sylvie-J** [community/community#53647] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi guys

My copilot stopped working a day or two ago and hasn't come back. I'm using vs code as my IDE. 

I've had a look at the logs and it just gives me: 
<img width="384" alt="image" src="https://user-images.githubusercontent.com/47174346/234163898-4fc18f1f-b27b-41b4-b11e-494bedd87001.png">

I've logged out of github and logged back in to check authentication and rolled back copilot, none of which worked. 

Any suggestions how to get it fixed?

**Comment by kobenauf** [comment]
This may or may not help you.  I was using VSC, which I launch from WSL2, e.g. `code .`.  Copilot used to work but started reporting 'inactive' as you show.  I went to [github's ssh keys](https://github.com/settings/keys), and added my standard rsa public key.  I restarted VSC, and then copilot started working.  What's strange is it had been working, and I have had keys set up and working, but maybe jumping on and off vpn somehow messed something up?  Not likely, but not sure what broke it to start with.

**Comment by mrcslws** [comment]
I ran into this issue, and finally found the culprit: my workspace had a local settings.json that was overriding the global GitHub Copilot setting. This did not show up in most UI, it just quietly prevented GitHub Copilot from working, even though this button was enabled:

<img width="29" alt="image" src="https://user-images.githubusercontent.com/364113/234381613-293e931a-20d7-4919-91ee-2b81f2e24ab7.png">

Arguably this is a bug in GitHub Copilot; that button should not be lit up when Copilot is disabled for the current workspace.

This workspace override is shown subtly in Settings:

![image](https://user-images.githubusercontent.com/364113/234381558-83c82ce2-aba9-425b-913d-0209d93ca767.png)

In `[project_dir]/.vscode/settings.json` I found the culprit:

```json
{
    "github.copilot.inlineSuggest.enable": false
}
```

When I erased this override, everything started working immediately.

**Reply by Sylvie-J** [reply]
Thanks, this was not the issue, but has lead me to the problem. It seems as if this was set to false: 
<img width="245" alt="image" src="https://user-images.githubusercontent.com/47174346/234462703-041580f7-919f-44d9-9a85-9148294f4139.png">

When set to true, it immediately started working.
Thanks for your help, I would've searched for hours!

**Comment by jack541** [comment]
I encountered the issue of "copilot is inactive",  but it is not the same cause as mentioned above.
![image](https://github.com/community/community/assets/25978072/71958221-520c-44a5-849c-cd19f47a668d)

I have also checked the configuration file, and all the "enable" settings are set to "true". 
![image](https://github.com/community/community/assets/25978072/28674ea0-a24a-491b-b605-a34c1146a154)
![image](https://github.com/community/community/assets/25978072/8aca7980-f488-4f6d-b421-737888e7108e)
![image](https://github.com/community/community/assets/25978072/63f10084-8274-46f6-928f-186d340eea71)

Version info  are 
vs code 
Version: 1.79.0-insider (Universal)
Commit: 05ae089c791aa5c8ec05abd5687c9a31517579bc
Date: 2023-06-02T09:42:42.311Z (2 hrs ago)
Electron: 22.5.5
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.1.0

Github Copilot Version:  v1.88.132


What else can I do to solve this problem? Thx!

**Reply by FeSeason** [reply]
Same question. 😭

### Copilot Setting
<img width="973" alt="image" src="https://github.com/community/community/assets/12840843/e650446e-df1f-46c5-90f1-2f9d9459a227">

### VsCode Copilot Output
```shell
[INFO] [copilotIgnore] [2023-06-04T02:31:18.084Z] inactive
```

### VsCode Version
```shell
Version: 1.78.2 (Universal)
Commit: b3e4e68a0bc097f0ae7907b217c1119af9e03435
Date: 2023-05-10T14:44:45.204Z
Electron: 22.5.2
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin x64 21.6.0
Sandboxed: Yes
```

**Comment by jack541** [comment]
The copilot's shows inactive, but the code suggestion works, why? 
![image](https://github.com/community/community/assets/25978072/6cb25130-59b6-460e-95fa-287ad70bdb86)
![image](https://github.com/community/community/assets/25978072/af38449e-3fee-40bf-b680-7e97a96bd86e)

**Reply by davecheney** [reply]
The `copilotIgnore` feature is inactive, not copilot itself.

**Comment by davecheney** [comment, marked as answer]
The `copilotIgnore` feature is inactive, not Copilot itself.

This is a debugging message which is errantly emitted. I believe it has been suppressed in newer versions of the extension.
